statsmodels/genmod/families/links.py

```diff
@@ -586,6 +586,149 @@ class log(Log):
     pass
 
 
+class LogC(Link):
+    """
+    The log-complement transform
+
+    Notes
+    -----
+    call and derivative call a private method _clean to trim the data by
+    machine epsilon so that p is in (0,1). logc is an alias of LogC.
+    """
+
+    def _clean(self, x):
+        return np.clip(x, FLOAT_EPS, 1)
```

Comment:
we need < 1 to avoid log(0)
if we define it for binomial, then we should use the same as for Logit
return np.clip(p, FLOAT_EPS, 1. - FLOAT_EPS)

Reply:
Agreed. I will make the change.

Reply:
Fixed and rebased.